packages/terra-worklist-data-grid/tests/wdio/worklist-data-grid-spec.js

```diff
@@ -28,98 +28,95 @@ Terra.describeViewports('WorklistDataGrid', ['medium', 'large'], () => {
 
     it('arrow keys', () => {
       browser.keys(['Tab']
-        .concat(new Array(5).fill('ArrowDown'))
-        .concat(new Array(6).fill('ArrowRight'))
-        .concat(new Array(3).fill('ArrowUp'))
-        .concat(new Array(2).fill('ArrowLeft')));
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
+        .concat(new Array(1).fill('ArrowUp'))
+        .concat(new Array(1).fill('ArrowLeft')));
 
+      browser.pause(150);
       Terra.validates.element('arrow-key-navigation', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('up arrow key cannot navigate outside grid', () => {
       browser.keys(['Tab']
-        .concat(new Array(1).fill('ArrowDown'))
         .concat(new Array(3).fill('ArrowUp')));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-grid-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('left arrow key cannot navigate outside grid', () => {
       browser.keys(['Tab']
-        .concat(new Array(1).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowUp'))
-        .concat(new Array(3).fill('ArrowLeft')));
+        .concat(new Array(2).fill('ArrowLeft')));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-grid-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('control with home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Control', 'Home', 'Control']));
 
+      browser.pause(150);
       Terra.validates.element('first-non-column-header-cell-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('home key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(2).fill('ArrowDown'))
+        .concat(new Array(2).fill('ArrowRight'))
         .concat(['Home']));
 
+      browser.pause(150);
       Terra.validates.element('first-cell-in-row-selected', { selector });
       expect(browser.$$('[role="grid"] [tabIndex="0"]')).toBeElementsArrayOfSize(1);
     });
 
     it('end key', () => {
       browser.keys(['Tab']
-        .concat(new Array(4).fill('ArrowDown'))
-        .concat(new Array(3).fill('ArrowRight'))
+        .concat(new Array(1).fill('ArrowDown'))
         .concat(['End']));
```

Comment:
We can combine this into a single array since it's pretty small
```js
browser.keys(['Tab', 'ArrowDown', 'End']);
```

Reply:
I will make this update.